src/bootstrap/sanity.rs

```diff
@@ -100,8 +100,15 @@ pub fn check(build: &mut Build) {
         if build.config.ninja {
             // Some Linux distros rename `ninja` to `ninja-build`.
             // CMake can work with either binary name.
-            if cmd_finder.maybe_have("ninja-build").is_none() {
-                cmd_finder.must_have("ninja");
+            if cmd_finder.maybe_have("ninja-build").is_none()
+                && cmd_finder.maybe_have("ninja").is_none()
+            {
+                panic!(
```

Comment:
nit: panic! is actually pretty bad experience on average, I'd prefer `eprintln!` and then `std::process::exit` for this kind of message.

Reply:
I used it because it's what `must_have` already uses. I can change it though.